CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Linkage Checker Enforcer Rule Change Log
 
+## 1.2.0
+* Linkage Checker takes an exclusion file to filter out known linkage errors
```

Comment:
do we need to update the docs for this?

Reply:
Updated document (removed artifact part)
https://github.com/GoogleCloudPlatform/cloud-opensource-java/wiki/Linkage-Checker-Exclusion-File